system/lib/compiler-rt/lib/asan/asan_emscripten.cpp

```diff
@@ -74,7 +76,7 @@ INTERCEPTOR(int, pthread_create, void *thread,
     StopInitOrderChecking();
   GET_STACK_TRACE_THREAD;
   int detached = 0;
-  if (attr)
+  if (attr && attr != __ATTRP_C11_THREAD)
```

Comment:
where is this value set? I don't understand what case is being handled here.

Reply:
Sorry I thought I had explained already.    Musl implements C11 threads on top of pthreads and uses this special value for the pthread attr pointer to mean "this is a C11 thread".   In this case attr is not a pointer at all bug a magic constant so we should not try to dereference it.  Its defined in `pthread_impl.h` which sadly not easy/possible to include from here.